### ISSUE TITLE
Unsupport CPU op: ReduceL2 in this librknnrt.so

### ISSUE THREAD

**Issue by AlfredFang** [rockchip-linux/rknn-toolkit2#300]
The README states that ReduceL2 operator is supported and inference is successful on RKNN-Toolkit2, but it throws an error during inference on RKNNRT, “Unsupport CPU op: ReduceL2 in this librknnrt.so”.

**Comment by prttyflvck0** [comment]
@AlfredFang have you solved this problem? I am facing the same issue.

**Comment by AlfredFang** [comment]
> @AlfredFang have you solved this problem? I am facing the same issue.

I replaced this operator with ReduceSum and Pow operators.

**Comment by prttyflvck0** [comment]
> > @AlfredFang have you solved this problem? I am facing the same issue.
> 
> I replaced this operator with ReduceSum and Pow operators.

 @AlfredFang Can you please share the code snippet of how you managed to do this? Many thanks in advance!

**Comment by AlfredFang** [comment]
> > > @AlfredFang have you solved this problem? I am facing the same issue.
> > 
> > 
> > I replaced this operator with ReduceSum and Pow operators.
> 
> @AlfredFang Can you please share the code snippet of how you managed to do this? Many thanks in advance!

In my case, I use the ReduceL2 operator to implement L2 normalization, and the following function is used instead.
```
def normalize(x, p=2.0, dim=1, eps=1e-12):
    norm = torch.pow(torch.sum(torch.pow(x, p), dim=dim, keepdim=True), 1/p).clamp(min=eps)
    return x / norm
```
